department-of-veterans-affairs/caseflow
# Remove admin/attorney buttons & labels from the JudgeTeam management page

Resolves #13992 

### Description
Removes all labels, buttons and all function calls associated with the _changing_ of the admin and decision drafting attorney status from the judge team page.  Additionally, the Remove from team button is no longer available on the judge team lead.

This PR also adds button headings & descriptions for the other orgs pages as they did not exist previously.

### Acceptance Criteria
- [x] JudgeTeam Team Management pages do not show admin toggle button
- [x] JudgeTeam Team Management pages do not show remove user button for the judge
- [x] JudgeTeam Team Management Decision Drafting Attorney button is instead a label
- [x] JudgeTeam Team Management explanatory text is updated
~- [ ] Other organization Team Management pages are unaffected~
- [x] Judge Team specific changes are locked to the `judge_admin_scm` FeatureToggle

### Testing Plan
1. Go to http://localhost:3000/organizations/bvaaabshire/users as BVAAABSHIRE
2. Make sure the buttons for enable/disable decision drafting and add/remove admin rights no longer show
3. Make sure that the Remove from team button is not present on the judge team lead
4. Check that the Remove from team button still works properly on the other users

### User Facing Changes

#### Judge Team management page
Before | After
-- | --
<img width="1244" alt="Screen Shot 2020-07-29 at 3 14 35 PM" src="https://user-images.githubusercontent.com/18618189/88842951-53488e00-d1ae-11ea-8c96-9fc88ca0ff51.png"> | <img width="1250" alt="Screen Shot 2020-07-31 at 1 25 30 PM" src="https://user-images.githubusercontent.com/18618189/89060790-9848fd80-d331-11ea-81e7-59f2171e3baa.png">

#### Other org management pages
Before | After
-- | --
<img width="1252" alt="Screen Shot 2020-07-31 at 1 32 20 PM" src="https://user-images.githubusercontent.com/18618189/89061404-a8151180-d332-11ea-9fc0-7cfec4ac50cd.png"> | <img width="1245" alt="Screen Shot 2020-07-31 at 1 32 39 PM" src="https://user-images.githubusercontent.com/18618189/89061421-ad725c00-d332-11ea-96bc-e97ee8c9e596.png">
